optuna_dashboard/_app.py

```diff
@@ -449,6 +451,60 @@ def save_trial_note(study_id: int, trial_id: int) -> dict[str, Any]:
         response.status = 204  # No content
         return {}
 
+    @app.get("/csv/<study_id:int>")
+    def download_csv(study_id: int) -> BottleViewReturn:
+        # Create a CSV file
+        summary = get_study_summary(storage, study_id)
+        if summary is None:
+            response.status = 404  # Not found
+            return {"reason": f"study_id={study_id} is not found"}
+        trials = get_trials(storage, study_id)
+
+        param_names = []
+        user_attr_names = []
+        for trial in trials:
+            for param_name in trial.params.keys():
+                if param_name not in param_names:
+                    param_names.append(param_name)
+            for attr_name in trial.user_attrs.keys():
+                if attr_name not in user_attr_names:
+                    user_attr_names.append(attr_name)
+
+        param_names_heading = [f"Param {x}" for x in param_names]
+        user_attr_names_heading = [f"UserAttribute {x}" for x in user_attr_names]
+        value_heading = ["Value"]
```

Comment:
It is very picky, but in my impression, I feel that `header` is more common for the column name rather than `heading`.

Reply:
I see. I revised the code.